src/current/cockroachcloud/connect-to-an-advanced-cluster.md

```diff
@@ -54,7 +54,9 @@ Private connectivity can be configured only after a cluster is created.
     - Set **Target** to **Published service**.
     - Set **Target service** to the value you copied from CockroachDB {{ site.data.products.cloud }} Console. If the endpoint's configured target service does not match, validation will fail.
     - Provide a value for **Endpoint name**. You will need to provide this **Endpoint name** when connecting to the cluster using the command line, a connection string, or connection parameters.
-    - If it is not enabled, enable the Service Directory API, click **Enable global access**, and create a namespace in each region where your cluster is deployed.
+    - If it is not enabled, enable the Service Directory API. Click **Enable global access**, and create a namespace in each region where your cluster is deployed.
+
+        When Service Directory and Global access are enabled, GCP automatically maps your private endpoint IPs to approved DNS names. CockroachDB {{ site.data.products.cloud }} requires DNS names in your connection information to match the approved pattern in your cluster’s certificates. Replace `<ENTER-ENDPOINT-NAME>` in your connection information with the assigned **Endpoint name** used in the previous step.
```

Comment:
It looks like we have not introduced `<ENTER-ENDPOINT-NAME>` at this point, and the user is still configuring their private endpoint. Should we remove the last sentence?

Reply:
Hmm, @devarshishah3 as per @jaylim-crl's comment above, are you referring to entering the endpoint name in the CockroachDB Cloud Console here? If so, I can clarify.

Reply:
<ENTER-ENDPOINT-NAME> is required when the user clicks the CONNECT modal and has to composes the connection string